nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -1,42 +1,36 @@
 #! @python3@/bin/python3 -B
-import argparse
-import shutil
-import os
-import sys
-import errno
-import subprocess
+
+from argparse import ArgumentParser
```

Comment:
please avoid `from` imports so it is clear directly in code where something comes from.